### PR TITLE
Improve commit message for preparation commit

### PR DESCRIPTION
**Pull request by bogdandrutu**
This sounds better since the modset represents a group of modules.

E.g. "Prepare stable modules for version v1.0.0-rc1"

Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

**Review by dmitryax — APPROVED** [review]

**Review by MrAlias — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Review by Aneurysm9 — APPROVED** [review]

**Review by XSAM — APPROVED** [review]

**Comment by MrAlias** [comment]
@bogdandrutu it looks like your last commit has an incorrect signature. Can you resign and push?

File: multimod/internal/prerelease/prerelease.go [hunk 1]
```diff
@@ -23,7 +23,6 @@ import (
 	"regexp"
 	"strings"
 
-	"github.com/go-git/go-git/v5"
```
Comment (MrAlias) [inline multimod/internal/prerelease/prerelease.go:26]: This shouldn't have been removed.

Comment (pellared) [reply multimod/internal/prerelease/prerelease.go:26]: Addressed https://github.com/open-telemetry/opentelemetry-go-build-tools/pull/210/commits/8606050c7b23c2f1a9a44c4910751e40856698d9

**Comment by pellared** [comment]
Closing as stale.

**Pull request closed without merging**